### ISSUE TITLE
Forbid kwarg unpacking in class definition

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1754]
# Rule request

## Thesis

We need to forbid code like this:

```python
class Some(Base, **kwargs):
    ...
```

But, this code is valid:

```python
class Some(Base, arg=1):
    ...
```

## Reasoning

Dynamic class generation with unknown values is really bad!

**Comment by StutiV** [comment]
Hello, @sobolevn Can you assign me this issue? I'd like to contribute. Thanks.

**Comment by sobolevn** [comment]
Thanks a lot!

**Comment by StutiV** [comment]
Thank you, @sobolevn ! Can you please elaborate the issue? It'd be really helpful.

**Comment by sobolevn** [comment]
@StutiV what help do you need? I would love to answer speicific questions 🙂

**Comment by StutiV** [comment]
@sobolevn  Which file should I be making these changes in?

**Comment by sobolevn** [comment]
@StutiV it is probably `ast/classes.py`. Feel free to choose the best visitor for the job.
https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/visitors/ast/classes.py

Probably `WrongClassVisitor`

**Comment by willtkwon** [comment]
Hi, can I work on this?

**Comment by StakanAgan** [comment]
@sobolevn can you assign this issue for me?

**Comment by sobolevn** [comment]
@StakanAgan done! Thanks!

**Comment by sobolevn** [comment]
For example, this is fine:

```python
from pydantic import BaseModel, Extra

class Model(BaseModel, extra=Extra.forbid):
    a: str
```

This is not (but will work the same way):

```python
from pydantic import BaseModel, Extra

kw = {'extra': Extra.forbid}
class Model(BaseModel, **kw):
    a: str
```

Source: https://pydantic-docs.helpmanual.io/usage/model_config/
